Makefile

```diff
@@ -116,64 +96,40 @@ cf-login: ## Log in to Cloud Foundry
 	@echo "Logging in to Cloud Foundry on ${CF_API}"
 	@cf login -a "${CF_API}" -u ${CF_USERNAME} -p "${CF_PASSWORD}" -o "${CF_ORG}" -s "${CF_SPACE}"
 
-.PHONY: cf-deploy
-cf-deploy: ## Deploys the app to Cloud Foundry
-	$(if ${CF_SPACE},,$(error Must specify CF_SPACE))
-	$(if ${CF_APP},,$(error Must specify CF_APP))
-	cf target -o ${CF_ORG} -s ${CF_SPACE}
-	@cf app --guid ${CF_APP} || exit 1
-
-	# cancel any existing deploys to ensure we can apply manifest (if a deploy is in progress you'll see ScaleDisabledDuringDeployment)
-	cf cancel-deployment ${CF_APP} || true
-
-	# fails after 15 mins if deploy doesn't work
-	CF_STARTUP_TIMEOUT=15 cf push ${CF_APP} --strategy=rolling
-
-.PHONY: cf-deploy-api-db-migration
-cf-deploy-api-db-migration:
-	$(if ${CF_SPACE},,$(error Must specify CF_SPACE))
-	cf target -o ${CF_ORG} -s ${CF_SPACE}
-	make -s CF_APP=notifications-api generate-manifest > ${CF_MANIFEST_PATH}
-
-	cf push notifications-api --no-route -f ${CF_MANIFEST_PATH}
-	rm ${CF_MANIFEST_PATH}
-
-	cf run-task notifications-api --command="flask db upgrade" --name api_db_migration
-
 .PHONY: cf-check-api-db-migration-task
 cf-check-api-db-migration-task: ## Get the status for the last notifications-api task
 	@cf curl /v3/apps/`cf app --guid notifications-api`/tasks?order_by=-created_at | jq -r ".resources[0].state"
 
-.PHONY: cf-rollback
-cf-rollback: ## Rollbacks the app to the previous release
-	$(if ${CF_APP},,$(error Must specify CF_APP))
-	rm ${CF_MANIFEST_PATH}
-	cf cancel-deployment ${CF_APP}
+# .PHONY: cf-rollback
+# cf-rollback: ## Rollbacks the app to the previous release
+# 	$(if ${CF_APP},,$(error Must specify CF_APP))
+# 	rm ${CF_MANIFEST_PATH}
+# 	cf cancel-deployment ${CF_APP}
```

Comment:
[suggestion] I think this can only cancel an in-progress deploy, not rollback once it is complete and we find a problem, so I think I prefer deleting it and starting from scratch if we want automated rollback ability